_posts/fr/newsletters/2024-08-23-newsletter.md

```diff
@@ -0,0 +1,239 @@
+  Ceci est implémenté dans LND (voir le Bulletin [#315][news315 blinded]) et est destiné à être utilisé
+  jusqu'à ce que le protocole [offers][topic offers] soit largement déployé dans le réseau.
+
+{% assign four_days_after_posting = page.date | date: "%s" | plus: 345600 | date: "%Y-%m-%d 14:30" %}
+{% include snippets/recap-ad.md when=four_days_after_posting %}
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="28553,30246,824,7540,7403,8943,1478,1533,1182,39,8751" %}
+[Core Lightning 24.08rc2]: https://github.com/ElementsProject/lightning/releases/tag/v24.08rc2
+[LND v0.18.3-beta.rc1]: https://github.com/lightningnetwork/lnd/releases/tag/v0.18.3-beta.rc1
+[moonsettler exfil1]: https://delvingbitcoin.org/t/non-interactive-anti-exfil-airgap-compatible/1081
+[wuille exfil1]: https://delvingbitcoin.org/t/non-interactive-anti-exfil-airgap-compatible/1081/3
+[wuille exfil2]: https://delvingbitcoin.org/t/non-interactive-anti-exfil-airgap-compatible/1081/7
+[news87 exfil]: /en/newsletters/2020/03/04/#proposal-to-standardize-an-exfiltration-resistant-nonce-protocol
+[news88 exfil]: /en/newsletters/2020/03/11/#exfiltration-resistant-nonce-protocols
+[maxwell exfil]: https://bitcointalk.org/index.php?topic=893898.msg9861102#msg9861102
+[news136 exfil]: /en/newsletters/2021/02/17/#anti-exfiltration
+[proton blog]: https://proton.me/blog/proton-wallet-launch
+[proton github]: https://github.com/protonwallet/
+[braidpool github]: https://github.com/braidpool/braidpool
+[cpunet post]: https://x.com/BobMcElrath/status/1823370268728873411
+[cpunet github]: https://github.com/braidpool/bitcoin/blob/cpunet/contrib/cpunet/README.md
+[lightningpub github]: https://github.com/shocknet/Lightning.Pub
+[taproot assets v0.4.0]: https://github.com/lightninglabs/taproot-assets/releases/tag/v0.4.0
+[sbm 0.1.0]: https://github.com/stratum-mining/benchmarking-tool/releases/tag/0.1.0
+[starkware tweet]: https://x.com/StarkWareLtd/status/1813929304209723700
+[bcs github]: https://github.com/Bitcoin-Wildlife-Sanctuary/bitcoin-circle-stark
+[news304 inquisition]: /fr/newsletters/2024/05/24/#bitcoin-inquisition-27-0
+[seedsigner website]: https://seedsigner.com/
+[seedsigner 0.8.0]: https://github.com/SeedSigner/seedsigner/releases/tag/0.8.0
+[floresta 0.6.0]: https://github.com/vinteumorg/Floresta/releases/tag/0.6.0
+[floresta blog]: https://medium.com/vinteum-org/floresta-update-simplifying-bitcoin-node-integration-for-wallets-6886ea7c975c
+[auto sys]: https://en.wikipedia.org/wiki/Autonomous_system_(Internet)
+[news290 asmap]: /fr/newsletters/2024/02/21/#amelioration-du-processus-de-creation-de-asmap-reproductible
+[news263 renepay]: /fr/newsletters/2023/08/09/#core-lightning-6376
+[alloy model]: https://alloytools.org/about.html
+[lnd linear]: https://github.com/lightningnetwork/lnd/blob/b7c59b36a74975c4e710a02ea42959053735402e/sweep/fee_function.go#L66-L109
+[news315 blinded]: /en/newsletters/2024/08/09/#lnd-8735
```

Comment:
Once we get changes to fr315 merged we can update this link, the rest looks good